ddl/util/util.go

```diff
@@ -325,3 +326,41 @@ func WrapKey2String(key []byte) string {
 	}
 	return fmt.Sprintf("0x%x", key)
 }
+
+const (
+	getRaftKvVersionSQL = "show config where type = 'tikv' && name = 'storage.engine'"
+	raftKv2             = "raft-kv2"
+)
+
+// IsRaftKv2 checks whether the raft-kv2 is enabled
+func IsRaftKv2(ctx context.Context, sctx sessionctx.Context) (bool, error) {
+	// Mock store does not support `show config` now, so we  use failpoint here
+	// to control whether we are in raft-kv2
+	failpoint.Inject("isRaftKv2", func(v failpoint.Value) (bool, error) {
```

Comment:
maybe intest.InTest

Reply:
move from https://github.com/pingcap/tidb/blob/79f00016016e9a1688613b7f4c8b5d830814fecb/store/gcworker/gc_worker.go#L840
so no change the original implementation